src/components/common/FilterableNamedEntityList.tsx

```diff
@@ -0,0 +1,117 @@
+import { Checkbox, debounce, FormControlLabel, FormGroup, Typography } from '@material-ui/core';
+import { makeStyles, Theme } from '@material-ui/core/styles';
+import { NamedEntity } from 'models/Common/types';
+import * as React from 'react';
+import { SearchableInput, SearchResult } from './SearchableList';
+import { useCommonStyles } from './styles';
+import { useSearchableListState } from './useSearchableListState';
+
+export const useStyles = makeStyles((theme: Theme) => ({
+  archiveCheckbox: {
+    whiteSpace: 'nowrap',
+  },
+  container: {
+    marginBottom: theme.spacing(2),
+    width: '100%',
+  },
+  noResults: {
+    color: theme.palette.text.disabled,
+    display: 'flex',
+    justifyContent: 'center',
+    marginTop: theme.spacing(6),
+  },
+  filterGroup: {
+    display: 'flex',
+    flexWrap: 'nowrap',
+    flexDirection: 'row',
+    margin: theme.spacing(4, 5, 2, 2),
+  },
+}));
+
+const NoResults: React.FC = () => (
+  <Typography className={useStyles().noResults} variant="h6" component="div">
+    No matching results
+  </Typography>
+);
```

Comment:
nit: we have 3 instances of this component in the codebase - should we define it separately and import, and move `No matching results` to `strings.ts`?
I'm fine with addressing it in a follow-up, but would like to make sure we have a task for it 😄

Reply:
Nice suggestion. For now I created a task for small cleanups: https://github.com/flyteorg/flyteconsole/issues/393 - we can pin it to the channel, or add to the doc in the channel.
There would be all small cleanups we know that need to be done. 
It would be nice to pick them up between tasks, or whenever you are in a right mind...

Reply:
In last update Carina did this change 🥳